<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>email</class>
 <widget class="QDialog" name="email">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#51556e;</string>
  </property>
  <widget class="QLineEdit" name="email_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
 border-width: 1px;
border-style: solid;
border-color: black;
  border-radius: 10px;
padding:0 8;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="email_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #FFFFFF;
background:#00BFFF;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 1px solid #A9A9A9;
border-top-left-radius: 7px;
border-top-right-radius: 7px;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background:#A9A9A9;
border: none;
border-top-left-radius: 7px;
border-top-right-radius: 7px;
}</string>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_96">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="objet_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
 border-width: 1px;
border-style: solid;
border-color: black;
  border-radius: 10px;
padding:0 8;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_94">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Objet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_95">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="description_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
 border-width: 1px;
border-style: solid;
border-color: black;
  border-radius: 10px;
padding:0 8;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
